src/gui/settingsdialog.cpp

```diff
@@ -137,8 +139,8 @@ SettingsDialog::SettingsDialog(ownCloudGui *gui, QWidget *parent)
     _actionGroupWidgets.insert(generalAction, generalSettings);
     _actionGroupWidgets.insert(networkAction, networkSettings);
 
-    foreach(auto ai, AccountManager::instance()->accounts()) {
-        accountAdded(ai.data());
+    foreach(auto account, AccountManager::instance()->accounts()) {
```

Comment:
Better to use a ranged for loop here

```suggestion
    for(const auto account : std::as_const(AccountManager::instance()->accounts())) {
```